io/subsystem/src/main/java/org/wildfly/extension/io/IOSubsystemTransformers.java

```diff
@@ -44,10 +47,19 @@ public String getSubsystemName() {
     public void registerTransformers(SubsystemTransformerRegistration registration) {
         ChainedTransformationDescriptionBuilder chainedBuilder = TransformationDescriptionBuilder.Factory.createChainedSubystemInstance(registration.getCurrentSubsystemVersion());
 
-        // Current 3.0.0 to 2.0.0, aka EAP 7.0.0
-        buildTransformers_2_0(chainedBuilder.createBuilder(registration.getCurrentSubsystemVersion(), VERSION_2_0));
+        // Current 3.0.0 to 2.0.0, aka EAP 7.1.0
```

Comment:
To @jmesnil's comment on #3107 - I think I've untangled the knot here but I'm not 100% sure.  A transformers expert should check this out to be sure.